onadata/apps/api/tests/viewsets/test_xform_viewset.py

```diff
@@ -3123,22 +3123,23 @@ def test_csv_export_with_win_excel_utf8(self):
                 '\ufeffage,\ufeffname,\ufeffmeta/instanceID,\ufeff_id,'
                 '\ufeff_uuid,\ufeff_submission_time,\ufeff_tags,\ufeff_notes,'
                 '\ufeff_version,\ufeff_duration,\ufeff_submitted_by,'
-                '\ufeff_total_media,\ufeff_media_count,'
-                '\ufeff_media_all_received\n\ufeff#age,,,,,,,,,,,,,\n29,'
+                '\ufeff_total_media,\ufeff_media_count,\ufeff_review_status,'
+                '\ufeff_review_comment,\ufeff_media_all_received\n'
+                '\ufeff#age,,,,,,,,,,,,,,,\n29,'
                 '\ufeffLionel Messi,'
                 '\ufeffuuid:74ee8b73-48aa-4ced-9072-862f93d49c16,%s,'
                 '\ufeff74ee8b73-48aa-4ced-9072-862f93d49c16,'
                 '\ufeff2013-02-18T15:54:01,\ufeff,\ufeff,\ufeff201604121155,'
-                '\ufeff,\ufeffbob,0,0,True\n' % data_id)
+                '\ufeff,\ufeffbob,0,0,n/a,n/a,True\n' % data_id)
```

Comment:
why is it n/a?  I thought we had agreed on it being not there?  I think that would mean that here we would just have nothing i.e. something like `,,,` for the two new fields